x-pack/plugins/apm/public/components/app/transaction_details/distribution/index.tsx

```diff
@@ -28,11 +29,25 @@ import { isErrorMessage } from '../../correlations/utils/is_error_message';
 
 const DEFAULT_PERCENTILE_THRESHOLD = 95;
 
+type Selection = [number, number];
+
+// Format the selected latency range for the "Clear selection" badge.
+// If the two values share the same unit, it will only displayed once.
+// For example: 12 - 23 ms / 12 ms - 3 s
+export function getFormattedSelection(selection: Selection): string {
+  const from = getDurationFormatter(selection[0])(selection[0]);
+  const to = getDurationFormatter(selection[1])(selection[1]);
```

Comment:
Not 100% sure but I seem to recall this is equivalent:

```suggestion
  const from = asDuration(selection[0]);
  const to = asDuration(selection[1]);
```

Reply:
Oh, sorry. You don't want the formatted value back.